Server/JokesOnYou.Web.Api/Repositories/UserRepository.cs

```diff
@@ -35,16 +42,42 @@ public async Task<User> CreateUserAsync(UserRegisterDTO userRegisterDTO)
                     message.AppendLine(error.Description);
                 }
 
-                throw new UserRegisterException(message.ToString());
+                throw new UserRegisterException(message.ToString(), new ArgumentException());
             }
         }
 
         public Task DeleteUserAsync(User user) => _userManager.DeleteAsync(user);
 
-        public Task<User> GetUserAsync(string id) => _userManager.FindByIdAsync(id);
+        public async Task<UserReplyDTO> GetUserReplyAsync(string id)
+        {
+            //Possibilities 
+            //var user = await _userManager.FindByIdAsync(id);
+            //var user = await _userManager.Users.FirstOrDefaultAsync(u => u.Id == id);
+            var user = await _userManager.FindByIdAsync(id);
+
+            return _mapper.Map<UserReplyDTO>(user);
+        }
+        public async Task<User> GetUserAsync(string id)
```

Comment:
remove the async and change to oneliner.

Reply:
its ok to use the async
https://github.com/Code2Gether-Discord/JokesOnYou/issues/116